contrib/babelfishpg_common/src/varchar.c

```diff
@@ -922,7 +922,11 @@ varchar2numeric(PG_FUNCTION_ARGS)
 	char	   *str;
 
 	str = varchar2cstring(source);
-	result = DatumGetNumeric(DirectFunctionCall1(numeric_in, CStringGetDatum(str)));
+	/* Updating numeric_in call with additional expected arguments `Elem type Oid` and `typmod` - BABEL-5129 */
+	result = DatumGetNumeric(DirectFunctionCall3(numeric_in, 
+        CStringGetDatum(str), 
```

Comment:
nit: please indent the arguments similar to the example below
![Screenshot 2024-09-19 at 12 08 48 PM](https://github.com/user-attachments/assets/d2e0d285-ad4f-4298-bd11-8de3baea5d5e)